### PR TITLE
visa payment added

### PR DESCRIPTION
**Pull request by masterpradeep**
added visa payment in application please review code and do the needfull

### CODE REVIEW

File: demo.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-Git is a great tool
```
Comment (viru2097) [inline demo.txt:1]: Why did you remove

Comment (masterpradeep) [reply demo.txt:1]: i chnegd logic so this line is not required
